src/Domain/Zone.php

```diff
@@ -2,33 +2,42 @@
 
 namespace SandwaveIo\RealtimeRegister\Domain;
 
+use SandwaveIo\RealtimeRegister\Domain\Enum\DomainZoneServiceEnum;
+
 final class Zone implements DomainObjectInterface
 {
-    public string $template;
+    public string $service;
+
+    public ?string $template;
 
     public ?bool $link;
 
-    private function __construct(string $template, ?bool $link)
+    private function __construct(string $service, ?string $template, ?bool $link)
```

Comment:
Minor: Add public to the constructor params so you don't have to write out the properties and set the value in the constructor.

```suggestion
    private function __construct(public string $service, public ?string $template, public ?bool $link)
```

Reply:
done